HelpSource/Classes/LevelComp.schelp

```diff
@@ -0,0 +1,154 @@
+TITLE:: LevelComp
+summary:: Level compensation logic for Splay and SplayAz
+categories:: Panning
+related:: Classes/Splay, Classes/SplayAz, Guides/Level_Compensation
+
+DESCRIPTION::
+Mixing and Panning multichannel signals often requires link::Guides/Level_Compensation::.
+LevelComp provides a logic for this that is used internally in the link::Classes/Splay:: and link::Classes/SplayAz:: UGens.
+Using the code::levelComp:: argument of code::Splay/Az::, it calculates a level compensation factor applied to the input signals, depending on the number of output channels.
+
+CLASSMETHODS::
+
+METHOD:: new
+Calculate a scaling factor based on the code::levelComp:: argument and the number of channels to be mixed.
+
+ARGUMENT:: levelComp
+a flag or numeric value:
+
+list::
+##code::true:: : equal power level compensation (default)
+##code::false:: : off / no level change,
+##float between code::0.0:: and code::1.0:: : smooth tuning of levelComp factor:
+list::
+##code::0.0:: is none -> factor 1
+##code::0.5:: is equal power -> factor (1/n).squared
+##code::1.0:: is equal amplitude -> factor (1/n)
+::
+::
+
+ARGUMENT:: rate
+the rate for which to calculate - \audio or \control.
+
+ARGUMENT:: n
+the number of channels being mixed and scaled.
+
+returns::
+the levelComp factor by which to multiply the output signals.
+
+EXAMPLES::
+
+code::Splay:: and code::SplayAz:: use code::LevelComp:: internally via the code::levelComp:: argument, and obtaining rate and number of channels from code::Splay/Az::.
```

Comment:
minor grammar note:
"and obtain the rate..."

Reply:
yes